.eslintrc.js

```diff
@@ -22,22 +22,22 @@ module.exports = {
   },
   plugins: ['@typescript-eslint', 'import', 'react'],
   rules: {
-    semi: 'error',
-    quotes: [1, 'single'],
-    'no-unused-vars': 'off',
-    '@typescript-eslint/no-unused-vars': ['error'],
-    'import/extensions': 'off',
-    'import/prefer-default-export': 'off',
-    'object-curly-newline': 'off',
-    'class-methods-use-this': 'off',
-    'no-shadow': 'off',
-    'no-console': 'off',
-    'arrow-body-style': 'off',
-    'no-useless-constructor': 'off',
+    semi: 2,
+    quotes: [1, 'single', 'avoid-escape'],
+    'no-unused-vars': 0,
+    '@typescript-eslint/no-unused-vars': 2,
+    'import/extensions': 0,
+    'import/prefer-default-export': 0,
+    'object-curly-newline': 0,
+    'class-methods-use-this': 0,
+    'no-shadow': 0,
+    'no-console': 0,
+    'arrow-body-style': 0,
+    'no-useless-constructor': 0,
     'no-sparse-arrays': 0,
     'no-inner-declarations': 0,
-    'no-use-before-define': 'off',
-    '@typescript-eslint/no-use-before-define': ['error'],
+    'no-use-before-define': 0,
+    '@typescript-eslint/no-use-before-define': 2,
```

Comment:
Why numbers are better than semantic text?

Reply:
I just want to merge them. Only one style needs to exist. @neoddish